packages/build-tools/src/steps/utils/expoUpdates.ts

```diff
@@ -35,9 +35,8 @@ export async function configureEASUpdateIfInstalledAsync({
   const expoUpdatesPackageVersion =
     await getExpoUpdatesPackageVersionIfInstalledAsync(workingDirectory);
   if (expoUpdatesPackageVersion === null) {
-    throw new Error(
-      `Cannot configure Expo Updates because the expo-updates package is not installed.`
-    );
+    logger.info('Expo Updates is not installed, skipping configuring Expo Updates.');
```

Comment:
Should we have some internal parameter that allows this to skip this only if run from within a function group? Like, if a user tries to configure updates in a manually written workflow, should we error?

Reply:
If the function group could dynamically define next step it would be ideal, but much more complicated.

Reply:
yeah, I agree

Reply:
Added new input to solve for this